app/components/conceptView/conceptView.controller.js

```diff
@@ -8,10 +8,10 @@
  * graphic logo is a trademark of OpenMRS Inc.
  */
 ConceptViewController.$inject = ['$scope', 'concept', 'serverLocales',
-                                'conceptsService', '$routeParams', 'conceptMappingsService'];
+                                'conceptsService', '$routeParams', 'conceptMappingsService', '$rootScope', '$translate'];
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2762346701/issues/source?bid=3196969&fileBranchId=3360987#l11)